app/assets/stylesheets/cart.css.scss

```diff
@@ -1,30 +1,78 @@
 .cart {
-    background: white;
+  background:white;
 }
 
-.cart-picture {
-    background: transparent image-url("checkout.jpg") center;
+.cart h3 {
+  padding-top:10px;
+  text-align:center;
+  font-weight: 200;
+  font-size: 42px;
 }
 
 .cart-picture {
-    height: 375px;
+  height: 200px;
+  background: transparent image-url("checkout.jpg") center;
+  box-shadow: 0px 10px 20px rgba(0,0,0,.3);
+}
+
+.cart-container {
+  height: 445px;
+  margin-right:auto;
```

Comment:
Properties should be ordered height, margin-left, margin-right<br>Colon after property should be followed by one space